content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md

```diff
@@ -74,7 +74,7 @@ Si está utilizando una computadora para verificar su identidad, puede cambiar a
       image='/assets/img/help/id-dos-and-donts/id-dont-white-background.png'
       bordered_image=true
       do_dont_text="NO"
-      text='Utilizar un fondo blanco'
+      text='NO use un fondo de color blanco.'
```

Comment:
I was looking at the diffs and noticed: We updated the spanish and french versions to say "Don't use a white background", but the english version still says `text='Use a white background'`

Reply:
Ah, good catch, Charley!

I am curious what you think @kellular . The TLDR is, should the word "No" be repeated in all languages or none?


Full explanation:

I may have been confused about the figma vs the translations, and parsing out what is code vs text. The figma doesn't repeat the word `NO` for English:

![image](https://github.com/GSA-TTS/identity-site/assets/35475380/411274c6-0efe-4134-9030-d52088f7ae9e)

My changes would double the word "NO" in Spanish and French:

![image](https://github.com/GSA-TTS/identity-site/assets/35475380/81f27adb-768c-4089-9547-9cf295d94a79)

When I did the Spanish and French, the "NO's" were there in the text for the translations. When I looked at the code, it seemed to me that the first "NO"s were icons (without going into too much detail - but in Charley's quoted code you can kind of see why). So I thought maybe we wanted to repeat the word in text copy for accessibility or clarity.

I should have asked though, now that I'm looking back on it I can see that the red "NO" is in fact text, not an icon. I _think_ the answer is that I should delete the repeat word from Spanish and French, but I want to be sure.

Reply:
This is a great catch, and sorry all -- I can see how reviewing this content between a few different artifacts has made the updates hard to decipher :/ 

> delete the repeat word from Spanish and French, but I want to be sure.

Yes this is correct

Reply:
No worries, I should have asked instead of going with my assumptions! This should be fixed now. Thanks for the confirmation!